src/modules/standard-clauses/dto/create-standard-clause.dto.ts

```
@@ -1,5 +1,20 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+import {
+  IsNotEmpty,
+  IsOptional,
+  IsString,
+  IsNumber,
+  Max,
+  Min,
+} from 'class-validator';
+import {
+  IsNotEmpty,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'IsNotEmpty'. Consider to delete it or rename it.

'IsNotEmpty' is defined here:


(lint/suspicious/noRedeclare)